gulpfile.js

```diff
@@ -0,0 +1,49 @@
+var gulp = require('gulp');
```

Comment:
This file has a mix of tabs and spaces. Let's go with 2 spaces.

Also a mix of single and double quotes. Let's use double.